packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -143,6 +143,20 @@ of a new version resource. You can specify options for this version through the
 > of code providers (such as `lambda.Code.fromBucket`) require that you define a
 > `lambda.Version` resource directly since the CDK is unable to determine if
 > their contents had changed.
+>
+> A workaround is to create a new function version whenever the stack is updated.
+> This makes sure your lambda function always has the latest code. One strategy
+> is to set an environment variable with a value that is guaranteed to be different
+> from one synthesis to the next.
```

Comment:
I don't think this is a strategy we can recommend here. 
We've strived to ensure that synthesizing an unchanged CDK code twice (including unchanged dependencies) should yield the same CloudFormation stack. We cannot say the opposite in the official documentation.

How about changing the wording to say, update this value whenever the code hosted on the s3 object has changed?

Reply:
I think that works.